docs/appendices/release-notes/unreleased.rst

```diff
@@ -58,6 +58,9 @@ None
 Fixes
 =====
 
+- Fixed an issue in the PostgreSQL wire protocol that would cause
+  de-serialization of arrays to fail if they contained unquoted strings
+  including digits.
```

Comment:
doesn't this only affect unquoted strings **starting** with a digit?
```suggestion
  starting with digits.
```

Reply:
Right, updated it :+1: